### ISSUE TITLE
Accessibility fixes for specs.canonical.com

### ISSUE THREAD

**Issue by marinacastejon** [canonical/specs-v1.canonical.com#67]
- Page hierarchy (there should be an H1 or an accessible alternative for screen readers)
- Skip to content button. Currently when navigating with a keyboard, the focus goes through the filters before reaching the main content (specs). The focus by clicking this "Skip to content button" should jump to the search field. 
- Language missing from markup
- When side panel is open, the focus can still go behind. The focus should be trapped in the side panel.

**Comment by marinacastejon** [comment]
Last issue, focus trapped on side panel not fixed in this issue. will be fixed in [this issue](https://app.zenhub.com/workspaces/-workplace-engineering-tribe-609110e401afd80010276c3b/issues/canonical-web-and-design/specs.canonical.com/58).

**Comment by marinacastejon** [comment]
Reopening this issue as agreed during standup @andesol 
- Page hierarchy (there should be an H1 or an accessible alternative for screen readers) -> Using a SR (VoiceOver) I can only access the hidden H1 ("Canonical specifications") when I search for headings. Correct behaviour: it should be the first thing the screenreader reads on the page, but it's a minor issue. We can group all minor A11y issues in the feature and work on the. 
- Skip to content button -> Fixed
- Language missing from markup -> Fixed
- When side panel is open, the focus can still go behind. The focus should be trapped in the side panel. -> Will be fixed in another issue (see previous comment).
Thanks!
